lib/lndclient/LndClient.ts

```diff
@@ -87,6 +97,10 @@ class LndClient extends BaseClient {
     return this.identityPubKey;
   }
 
+  public getCltvDelta() {
```

Comment:
I still find this method unnecessary, and it still technically allows `cltvDelta` to be changed after the fact when from what I understand that shouldn't be possible. It's also just a deviation from the style/structure of the rest of the codebase I think. If you just declare `cltvDelta` as `public readonly cltvDelta = 0` you can read it from anywhere but never overwrite it and without the need for an extra method.

Reply:
OK